Habitica/src/triggers/TaskPointCalculation.trigger

```diff
@@ -0,0 +1,34 @@
+trigger TaskPointCalculation on Task__c (after insert, after update) {
+    for(Task__c task : Trigger.New) {
+        if (((Trigger.isInsert) || (Trigger.oldMap.get(task.Id).Status__c != task.Status__c )) && task.Status__c == 'Completed') {
+            RecordType rt = [SELECT Name FROM RecordType WHERE Id = :task.RecordTypeId Limit 1];
+            Contact employee = [select Name, Total_Points_Calculated__c from Contact where id = :task.Employee__c Limit 1];
```

Comment:
Define the variables 'rt' and empoyee outside the loop

Reply:
I put the variables rt and employee inside the loop because the Trigger.New variable receives and array of Task__c, its a precaution for executions in batches, but really the loop executes just one iteration.